feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts/progenesis_formatter.py

```diff
@@ -41,26 +56,19 @@ def convert_to_feature_csv(input_filename, output_filename):
         output_record = {}
 
         output_record["row ID"] = str(running_scan)
-        output_record["row m/z"] = str(mz)
-        output_record["row retention time"] = str(rt)
+        output_record["row m/z"] = float(mz)
+        output_record["row retention time"] = float(rt)
 
         output_record2 = {}
         for sample_name in sample_names:
             output_record2[sample_name + " Peak area"] = record[sample_name]
 
-        #Adding in special columns for CCS
-        if "CCS (angstrom^2)" in record:
-            output_record["CCS"] = record["CCS (angstrom^2)"]
-        if "dCCS (angstrom^2)" in record:
-            output_record["dCCS"] = record["dCCS (angstrom^2)"]
-
         #Adding extra columns that are not sample_names
         for key in record:
-            if key in sample_name:
+            if key in sample_names:   ###
                 continue
-
             try:
-                output_record[key] = record[key].encode("ascii", errors="replace")
+                output_record[key] = record[key]
```

Comment:
Does this not need an ascii cleanup anymore?

Reply:
I am not sure, but this was causing an issue for some metadata columns.

Reply:
Without cleaning up, I'm not sure we won't have an issue later, maybe here is better to to do an encode and ignore instead of replace.

Reply:
It works fine currently. I can do the change but it is on you if its brake ^^

Reply:
Ok actually I tried and it brakes the code. Maybe because I parse the table as utf-8.
The code works fine !